lib/wallaroo/core/keys/local_keys_file.pony

```diff
@@ -47,11 +47,13 @@ class LocalKeysFile
   =>
     _writer = Writer
     _filepath = fpath
+    @printf[I32]("SLF: LocalKeysFile async'ed with %s\n".cstring(), _filepath.path.cstring())
```

Comment:
Do we want to keep these `@printf` calls?

Reply:
No.  I'll drop them.